.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.6
```

Comment:
Helping to standardize the environment this gem is used in, based on the currently known use cases (HiMama).